tests/foreman/cli/test_satellitesync.py

```diff
@@ -113,7 +113,7 @@ class TestRepositoryExport:
 
     @pytest.mark.tier3
     def test_positive_export_complete_version_custom_repo(
-        self, default_sat, export_import_cleanup_module, module_org
+        self, target_sat, export_import_cleanup_function, module_org
```

Comment:
Shouldn't this be `export_import_cleanup_module` to keep it consistent with what was used before?